molgenis-dataexplorer/src/main/resources/js/dataexplorer.js

```diff
@@ -304,8 +305,18 @@ $.when($,
                 }
             });
 
+            $.get(molgenis.getContextUrl() + '/packageHref?entity=' + state.entity).done(function (data) {
+                if (data.href !== undefined) {
+                    $("#entity-package-path").removeClass('hidden');
+                    $("#entity-package-path").prop("href", data.href);
+                    $("#entity-package-path").html('(<span class="glyphicon glyphicon-home" aria-hidden="true"></span> / ' + data.fullLabel + ')');
+                } else {
+                    $("#entity-package-path").addClass('hidden');
+                }
+            });
+
             // get entity meta data and update header and tree
-            var entityMetaDataRequest = restApi.getAsync('/api/v1/' + state.entity + '/meta', {expand: ['attributes']}, function (entityMetaData) {
+            var entityMetaDataRequest = restApi.getAsync('/api/v1/' + state.entity + '/meta', {expand: ['attributes', 'package']}, function (entityMetaData) {
```

Comment:
Oehh, I like this

Reply:
it should go however, since it is a leftover from a failed attempt